api/routes/admin.js

```diff
@@ -17,30 +20,30 @@ router.get("/users/not-approved", async (req, res) => {
   }
 });
 
-router.put("/users/:id/set-is-approved", body('approve').isBoolean(), async (req, res) => {
-  const userId = req.params.id;
-  const isApproved = req.body.approve;
-
-  const { errors } = validationResult(req);
-  if (errors.length > 0) {
-    log.debug(`${req.originalUrl} failed req validation: ${JSON.stringify(errors, null, 2)}`);
-    return res.sendStatus(400);
-  }
-
-  try {
-    const user = await Users.findByIdAndUpdate(userId, { isApproved });
-
-    if (!user) {
-      log.debug(`${req.originalUrl} no user found with id: ${userId}`);
-      return res.sendStatus(404);
+router.put(
+  "/users/:id/set-is-approved",
+  reqBodyValidation(setIsApprovedReqSchema),
```

Comment:
maybe validateReqBody is a little more reader friendly? no big deal

```suggestion
  validateReqBody(setIsApprovedReqSchema),
```

Reply:
Updated 👍